diff -urN fungimol-0.5.1.org/Scene/SelectionManager.cpp fungimol-0.5.1/Scene/SelectionManager.cpp
--- fungimol-0.5.1.org/Scene/SelectionManager.cpp	Fri Mar  7 21:54:17 2003
+++ fungimol-0.5.1/Scene/SelectionManager.cpp	Sat Mar  8 14:18:50 2003
@@ -119,7 +119,7 @@
     // Don't try to inline this since it has a static variable.
     void animateFrame (Canvas *canvas, SceneGraph *sg, int index) const;
 
-    static const char *const staticClassId = "SelectionInvisibilityManager";
+    static const char *const staticClassId;
 
     const char *classId () const {
       return staticClassId;
@@ -131,6 +131,8 @@
     }
   };
 
+  const char *const SelectionInvisibilityMarker::staticClassId = "SelectionInvisibilityManager";
+
   void SelectionInvisibilityMarker::animateFrame (Canvas *canvas, SceneGraph *sg, int index) const {
     (void) canvas; (void) sg; (void) index;
     // Blink on and off with each frame.
